app/templates/components/promoted-group-card.hbs

```diff
@@ -13,7 +13,7 @@
         </span>
       </div>
     </span>
-    <button class="blue {{if (not this.authManager.currentUser.email) 'disabled'}} {{if this.device.isMobile 'mt-2' 'mt-4'}} ui fluid button" {{action 'follow'}}>
```

Comment:
Why remove this. This will remove the inactive color from the button while not logged in.

Reply:
We have to add alpha to the color of the button as discussed in meeting @pc-beast .